### ISSUE TITLE
Organize error messages better

### ISSUE THREAD

**Issue by cjmyers** [SynBioDex/SBOL-Validator#97]
Make it very clear in the validator when an error is a error (serious problem, red checkbox in the list), a completeness issue (blue circle, can be ignored, if you like by turning off completeness checking), or a best practice (this is off by default actually).

Maybe an output like:

We found W validation errors in your file, click here for more details.
These are serious errors with your SBOL file that cannot be ignored.

We found X completeness warnings in your file, click here for more details.
If you wish to ignore these in the future, select Allow incomplete documents.

We found Y URI compliance warnings in your file, click here for more details.
If you wish to ignore these in the future, select Allow non-compliant URIs.

We found Z best practice warnings in your file, click here for more details.
If you wish to ignore these in the future, de-select Check best practices.

**Comment by cjmyers** [comment]
Add this text to that above:

"These errors are typically due to a problem with the software that generated this SBOL file.  Therefore, you should report these errors to the developers of that software package."

Longer term, would be cool to have an email address field that the user could fill to automatically forward the errors to the developers along with the example file.

**Comment by zachzundel** [comment]
I'm happy to do this, but is there any way we could add better error passing to libSBOLj, specifcally some kind of structured output I don't have to build a parser for? 
JSON would work well, and there's lot of existing infrastructure in Java and Python (the backend language of the validator.) If not, even null-byte-terminated messages would help. I just think hand-written deserialization code is going to be a beast to maintain.

**Comment by cjmyers** [comment]
Yes, let's do this.  We can work out format when we meet Thursday.
